core/shared/src/main/scala/org/typelevel/log4cats/extras/ConsoleLogger.scala

```diff
@@ -0,0 +1,323 @@
+/*
+ * Copyright 2018 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.typelevel.log4cats.extras
+
+import cats.{~>, Show}
+import cats.effect.kernel.{Async, Ref}
+import cats.effect.std.Console
+import cats.syntax.all.*
+import org.typelevel.log4cats.SelfAwareStructuredLogger
+
+import java.io.{ByteArrayOutputStream, PrintStream}
+
+/**
+ * A simple logger that prints logs to standard error output.
+ *
+ * The intended use-case is for one-off scripts, examples, and other situations where a more
+ * fully-featured and performance-optimized logger would be overkill
+ *
+ * Log output format is roughly:
+ * {{{
+ * %level %name - %message
+ *   %ctx
+ *   %throwable
+ * }}}
+ */
+trait ConsoleLogger[F[_]] extends SelfAwareStructuredLogger[F] {
+
+  /**
+   * Because there is no `log4j` backend, the log level can be set directly
+   */
+  def setLogLevel(level: LogLevel): F[Unit]
```

Comment:
Are we sure that `setLogLevel` belongs to the Logger? As far as I can tell, that API usually corresponds to a Logger factory.

Reply:
On further reflection, they can set the log level when they're making it, so this probably isn't needed.